src/main/java/it/gov/pagopa/pu/migration/wf/activity/ingestion/organizations/OrganizationsMigrationFileTypeHandlerActivityImpl.java

```diff
@@ -35,7 +48,39 @@ protected MigrationFileTypeEnum getHandledMigrationFileType() {
 
   @Override
   protected MigrationFileResult handleRetrievedFiles(List<Path> retrievedFiles, Uploads ingestionFlowFileDTO) {
```

Comment:
can you rename this variable?
```suggestion
  protected MigrationFileResult handleRetrievedFiles(List<Path> retrievedFiles, Uploads upload) {
```